installer/database/migrations/migrate_0.5.9.5.sql

```
@@ -0,0 +1,176 @@
+    CONSTRAINT fk_lc_collana FOREIGN KEY (collana_id) REFERENCES collane(id) ON DELETE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+INSERT IGNORE INTO collane (nome)
+SELECT DISTINCT l.collana
+  FROM libri l
+ WHERE l.collana IS NOT NULL
+   AND l.collana != ''
+   AND l.deleted_at IS NULL;
+
+INSERT IGNORE INTO libri_collane (libro_id, collana_id, numero_serie, tipo_appartenenza, is_principale)
+SELECT l.id, c.id, l.numero_serie, 'principale', 1
```

SQLFluff (4.1.0)

[error] Reuse of column alias 'id' from line 158.

(AL08)